### PR TITLE
Improve table formatting in docs

### PR DESCRIPTION
**Pull request by discdiver**
Fix formatting on a number of docs tables. 
Find/replace suggests we should have them all updated now.
Addresses user feedback on table formatting.

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
  - If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [ ] If this pull request adds new functionality, it includes unit tests that cover the changes
- [ ] If this pull request removes docs files, it includes redirect settings in `mint.json`.
- [ ] If this pull request adds functions or classes, it includes helpful docstrings.

### CODE REVIEW

**Comment by discdiver** [comment]
Before:
Screenshot 2024-10-15 at 10 22 57 AM

After:
Screenshot 2024-10-15 at 10 24 05 AM

File: docs/3.0/develop/inputs.mdx [hunk 1]
```diff
@@ -767,17 +767,17 @@ python my_file_name greeter
...
+______________________________________________________________________
+| Your flow 'greeter' is being served and polling for scheduled runs  |
+|                                                                    |
+| To trigger a run for this flow, use the following command:          |
+|                                                                    |
+|         $ prefect deployment run 'greeter/send-receive'            |
+|                                                                    |
+| You can also run your flow via the Prefect UI:                      |
+| https://app.prefect.cloud/account/...(a URL for your account)      |
+|                                                                    |
```
Comment (desertaxle) [inline docs/3.0/develop/inputs.mdx:779]: Looks like there's a couple of `|` that are off by a space or two here.

Is it worthwhile to have CLI outputs like this in the docs, considering the effort required to keep it in sync with the actual output?

Comment (discdiver) [reply docs/3.0/develop/inputs.mdx]: Good catch. Updated. 

Good question. There are pros and cons to showing it. I don't have a strongly held preference.

File: docs/3.0/deploy/infrastructure-examples/serverless.mdx [hunk 2]
```diff
@@ -267,26 +267,26 @@ run this command for your particular cloud provider:
...
+    _____________________________________________________________________________________________
+    | Provisioning infrastructure for your work pool my-aci-work-pool will require:             |
+    |                                                                                           |
+    |     Updates in subscription Azure subscription 1                                          |
+    |                                                                                           |
+    |         - Create a resource group in location eastus                                      |
+    |         - Create an app registration in Azure AD prefect-aci-push-pool-app                |
+    |         - Create/use a service principal for app registration                             |
+    |         - Generate a secret for app registration                                          |
+    |         - Create an Azure Container Registry with prefix prefect                           |
```
Comment (zzstoatzz) [inline docs/3.0/deploy/infrastructure-examples/serverless.mdx:279]:
```suggestion
    |         - Create an Azure Container Registry with prefix prefect                          |
```

Comment (zzstoatzz) [inline docs/3.0/deploy/infrastructure-examples/serverless.mdx:279]:
```suggestion
    |         - Create an Azure Container Registry with prefix prefect                          |
```

**Review by zzstoatzz — APPROVED** [review]
lgtm except for the extra space on [L279](https://github.com/PrefectHQ/prefect/pull/15700/files#diff-586b35f4a95fe25db6581026f73f055b0576de6d9acdf9296041faf6fea619f6R279)

File: docs/3.0/deploy/infrastructure-examples/serverless.mdx (hunk 2 above)
Comment (discdiver) [reply docs/3.0/deploy/infrastructure-examples/serverless.mdx:279]: Weirdly there was one spot that renders as a straight line, although it is off by one in the file.

**Pull request merged**